packages/quick_actions/quick_actions_ios/example/ios/RunnerUITests/RunnerUITests.swift

```diff
@@ -51,46 +51,4 @@ class RunnerUITests: XCTestCase {
 
     XCTAssert(actionTwoConfirmation.exists)
   }
-
-  func testQuickActionWhenAppIsInBackground() {
```

Comment:
Why are you deleting this test?

Reply:
Sorry it was mistake, revert.

Reply:
Hi, my previous comment on adding XCUITest is not addressed. Are you having any issue writing this test?

Reply:
Hey, yes I am having an Issue with writing the test..

Reply:
Hi, in XCUITests, you shouldn't touch any code of quick action. You can follow similar examples in other XCUITests we have.

Reply:
So how can I create a XCUITests for that if eventually I can't get the type from the quick action I just pressed?

Reply:
you should update the ui based on the returned type, and verify that ui on the screen.